config/routes.rb

```diff
@@ -10,6 +10,7 @@
 
   resources :actors
   resources :actor_types, only: [:index, :show]
+  resources :actor_types_taxonomies, only: [:index, :show]
```

Comment:
endpoint should be `actor_type_taxonomies` for consistency 

(although I would have generally preferred to have `actortypes` and `actortype_taxonomies` as the `_` then always indicates a m:n-relationships between actortypes and taxonomies (likewise actortype is not a join table between actors and types) - regardless I realise I should have raised this earlier but if this is easy to change it would actually be great to change this everywhere (same for `measuretype`))

Reply:
please also see #57